lasy/utils/exp_data_utils.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+
+def find_com(img):
```

Comment:
Maybe this? We probably don't call it so often that we need a short name
```suggestion
def find_center_of_mass(img):
```